File: C++/2460.cpp
#include <iostream>
//https://www.acmicpc.net/problem/2460
int main()
{
  using namespace std;
  int in, out;
  // 변수 여러개를 초기화 할 때는 각각 초기화를 해주어야 쓰레기 값이 안들어간다
  int ans = 0, now = 0;
  for(int i = 0; i < 10; i++)
  {
    cin >> out >> in;
    now += (in - out);
    if(ans < now)
      ans = now;
  }
  cout << ans << endl;
  return 0;
}

File: Python/2460.py
from sys import stdin
//https://www.acmicpc.net/problem/2460
def main():
  ans, now = 0, 0
  for i in range(10):
    off, on = map(int, stdin.readline().split(" "))
    now += (on - off)
    if(ans < now):
      ans = now
  print(ans)

if __name__ == "__main__":
  main()